core/cmd/vrf_keys_commands.go

```diff
@@ -0,0 +1,277 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	"github.com/smartcontractkit/chainlink/core/services/signatures/secp256k1"
+	"github.com/smartcontractkit/chainlink/core/web/presenters"
+	"go.uber.org/multierr"
+
+	"github.com/pkg/errors"
+	"github.com/smartcontractkit/chainlink/core/utils"
+	"github.com/urfave/cli"
+)
+
+type VRFKeyPresenter struct {
+	JAID // Include this to overwrite the presenter JAID so it can correctly render the ID in JSON
+	presenters.VRFKeyResource
+}
+
+func (p VRFKeyPresenter) FriendlyDeletedAt() string {
+	if p.DeletedAt != nil {
+		return p.DeletedAt.String()
+	}
+	return "\n"
+}
+
+// RenderTable implements TableRenderer
+func (p *VRFKeyPresenter) RenderTable(rt RendererTable) error {
+	headers := []string{"Compressed", "Uncompressed", "Hash", "Created", "Updated", "Deleted"}
+	rows := [][]string{p.ToRow()}
+	renderList(headers, rows, rt.Writer)
```

Comment:
```suggestion
	renderList(headers, rows, rt.Writer)
	rt.Write([]byte("\n"))
```

Otherwise maybe change `renderList` to always write a new line at the end

Reply:
+1 makes sense